packages/zod-firebase/src/query/__tests__/query-specification.test.ts

```
@@ -1,4 +1,15 @@
-import { limit, orderBy, type Query, where } from '@firebase/firestore'
+import {
```

ESLint

[error] Resolve error: EACCES: permission denied, open '/myjCWIvxTT'
    at Object.writeFileSync (node:fs:2409:20)
    at l (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:13685)
    at createFilesMatcher (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:14437)
    at Object.resolve (/home/jailuser/git/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.3_eslint-plugin-import-x@4.15.1_@typescript-eslin_931e3c8d5baeee62667ad64678ef1db5/node_modules/eslint-import-resolver-typescript/lib/index.cjs:296:107)
    at resolveWithLegacyResolver (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.15.1_@typescript-eslint+utils@8.33.1_eslint@9.31.0_jiti@2.4.2__038eacbef045d199794e2aa3d08ecb9a/node_modules/eslint-plugin-import-x/lib/utils/legacy-resolver-settings.js:8:25)
    at file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.15.1_@typescript-eslint+utils@8.33.1_eslint@9.31.0_jiti@2.4.2__038eacbef045d199794e2aa3d08ecb9a/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:196:60
    at setRuleContext (/home/jailuser/git/node_modules/.pnpm/eslint-import-context@0.1.8_unrs-resolver@1.7.11/node_modules/eslint-import-context/lib/index.js:23:20)
    at fullResolve (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.15.1_@typescript-eslint+utils@8.33.1_eslint@9.31.0_jiti@2.4.2__038eacbef045d199794e2aa3d08ecb9a/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:196:30)
    at relative (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.15.1_@typescript-eslint+utils@8.33.1_eslint@9.31.0_jiti@2.4.2__038eacbef045d199794e2aa3d08ecb9a/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:207:12)
    at resolve (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.15.1_@typescript-eslint+utils@8.33.1_eslint@9.31.0_jiti@2.4.2__038eacbef045d199794e2aa3d08ecb9a/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:212:16)

(import-x/namespace)